### ISSUE TITLE
Please add new page with benchmark results

### ISSUE THREAD

**Issue by sirzooro** [skvadrik/re2c#149]
Please add new page with benchmark results. I did some on my own and it turned out that after adding `-s` or `-sb` generated code was worse than baseline, but with `-sbg` it was better (results are in #148). It would be nice if you could perform some benchmarks using various platforms and compilers, and put results somewhere.

It would be also nice to see how other lexers (e.g. flex) performs in comparison with re2c in equivalent test cases.

**Comment by skvadrik** [comment]
Yes, absolutely. Benchmarking is one of the most interesting things about re2c; the question has been asked a number of times on mailing lists and elsewhere.

It's hard to make representative benchmarks, so I took a gradual approach to the problem.

One of the first steps was adding skeleton (http://re2c.org/manual/features/skeleton/skeleton.html#benchmarks, `-S`, `--skeleton` option). Skeleton spares you the time for writing interface code: you can write regular expression specifications and re2c will generate everything else (including input data with very good coverage). Of course, skeleton preserves all code generation options, so that you can try it with `-s`, `-b`, `-g` or whatever.

The next step is to experiment with various regular expressions and options and put it all together on some website page.

**Comment by skvadrik** [comment]
I recently added infrastructure to run benchmarks and a web-page with the benchmark results: https://re2c.org/benchmarks/benchmarks.html. The current focus is on submatch extraction (there's no comparison for different re2c code generation options, like `-b` or `-g`), but it's a start.

**Comment by trofi** [comment]
Let's close as we now have a basic page with some numbers that we can extend.
